assets/js/admin/course-pricing-nudge-sidebar.js

```diff
@@ -0,0 +1,68 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters, addFilter } from '@wordpress/hooks';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { escapeHTML } from '@wordpress/escape-html';
+
+/**
+ * Course Pricing Nudge Sidebar component.
+ */
+
+const CoursePricingNudgeSidebar = () => {
+	/**
+	 * Filters to get description for pricing component.
+	 *
+	 * @since 4.1.0
+	 *
+	 * @hook  senseiCoursePricingDescription
+	 * @return {Object}
+	 */
+	const description = applyFilters( 'senseiCoursePricingDescription', {
+		text: __(
+			'Sell this course using WooCommerce - integrates with subscriptions, memberships, affiliates, and more.'
+		),
+		url:
+			'https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=course_pricing',
+	} );
+
+	return (
+		<PluginDocumentSettingPanel
+			name="sensei-course-theme"
+			title={ <>{ __( 'Pricing ', 'sensei-lms' ) }</> }
+		>
+			<div className="sensei-pricing-nudge">
+				<div className="sensei-pricing-nudge__content">
+					<p> { escapeHTML( description.text ) } </p>
+					<a
+						className="sensei-pricing-nudge__redirect-icon"
+						href={ description.url }
+						target="_blank"
+						rel="noreferrer"
+					>
+						{ __( 'Upgrade to Sensei Pro', 'sensei-lms' ) }
+					</a>
```

Comment:
Donna [mentioned](https://github.com/Automattic/sensei/pull/4816#discussion_r807117753) that we could use the [ExternalLink](https://developer.wordpress.org/block-editor/reference-guides/components/external-link/) component. Should simplify things. What do you think?

Reply:
Changed [here](https://github.com/Automattic/sensei/pull/4810/files#diff-0cdf94ac1cfb0a1e90adaef288fdb975f07f4c39ff34e0de2003256181b3ec93R35)